libcontainerd/client_local_windows.go

```diff
@@ -326,6 +326,13 @@ func (c *client) createWindows(id string, spec *specs.Spec, runtimeOptions inter
 	}
 	configuration.MappedPipes = mps
 
+	// Add any device assignments
+	if len(spec.Windows.Devices) > 0 {
+		for _, d := range spec.Windows.Devices {
```

Comment:
Should an error be returned here if assigned devices are specified on Windows builds lower than RS5, or is the check in oci_windows.go sufficient? A version check for named pipes is done in this function.

Reply:
I added the same errors in both places. I thought it would suffice in oci_windows.go but this seems to be safer overall.